medusa/providers/torrent/html/scenetime.py

```diff
@@ -131,9 +132,9 @@ def parse(self, data, mode):
                     continue
 
                 try:
-                    link = cells[labels.index('Name')].find('a')
-                    torrent_id = link['href'].replace('details.php?id=', '').split('&')[0]
-                    title = link.get_text(strip=True)
+                    link = cells[labels.index('Name')]
+                    torrent_id = link.find('a')['href'].replace('details.php?id=', '').split('&')[0]
+                    title = link.find('a').get_text(strip=True)
```

Comment:
Why this change?

Reply:
That way I can use the `link`, already defined before, for the lineage. And isn't defined again twice.

Reply:
I see it now, all good.